src/css/_header.scss

```diff
@@ -1,11 +1,31 @@
-header {
+@import "variables";
```

Comment:
I like how your created a partial file for a specific section of your page. It reminds me of how structuring works in React (from my understanding)